x-pack/plugins/endpoint/public/applications/endpoint/store/alerts/types.ts

```diff
@@ -0,0 +1,10 @@
+/*
```

Comment:
Could we move these types to `applications/endpoint/types.ts`? We are trying to keep all the public types in one file for now.